spt/features/visualizations/renderer/internal/mesh_renderer.cpp

```diff
@@ -83,7 +83,9 @@ namespace patterns
 	         "7467727-hl2",
 	         "55 8B EC 81 EC A0 00 00 00 83 3D ?? ?? ?? ?? 00");
 	PATTERNS(CSkyBoxView__DrawInternal,
-	         "5135-portal1",
+	         "EZ2-1.0.0+original",
+	         "55 8B EC 81 EC 28 01 00 00 A1 ?? ?? ?? ?? 33 C5 89 45 ?? 8B 45 ??",
+			 "5135-portal1",
```

Comment:
what's going on with this indenting over here

Reply:
My IDE is constantly trying to make up its mind over whether to do whitespaces or tabs, and the auto-format also produces some wacky results